Gems/Multiplayer/Code/Include/Multiplayer/Components/NetBindComponent.h

```diff
@@ -108,6 +119,7 @@ namespace Multiplayer
         void NotifyMigrationEnd();
         void NotifyServerMigration(HostId hostId, AzNetworking::ConnectionId connectionId);
         void NotifyPreRender(float deltaTime, float blendFactor);
+        void NotifyCorrection();
```

Comment:
I'm not sure if the difference between NotifyCorrection and NotifySyncRewindState may be ambiguous to new users. Maybe NotifyPreCorrection is clearer?

Reply:
One generally happens on the server, and one on the client.. I should add comments though.